modules/cnv_note_imports.adoc

```diff
@@ -0,0 +1,9 @@
+[[note-on-imports]]
+=== A note on importing disk images
+
+When you import a disk image into a PersistentVolumeClaim, the image is resized 
+to the size requested in the PVC. 
+
+After importing the image, you may need to perform a filesystem resize operation 
```

Comment:
s/may/might
s/filesystem/file system (While there's not an explicit entry in either, all the examples in both the IBM style guide and CCS glossary use two words.)